scripts/gen_profiles_solution.py

```diff
@@ -2233,6 +2233,9 @@ def applyWorkarounds(self):
         if 'VkPhysicalDeviceFragmentDensityMapOffsetPropertiesQCOM' in self.structs:
             self.structs['VkPhysicalDeviceFragmentDensityMapOffsetPropertiesQCOM'].members['fragmentDensityOffsetGranularity'].limittype = 'min,mul'
 
+        if 'VkPhysicalDeviceSchedulingControlsPropertiesARM' in self.structs:
+            self.structs['VkPhysicalDeviceSchedulingControlsPropertiesARM'].members['schedulingControlsFlags'].limittype = 'min'
```

Comment:
@christophe-lunarg Is this correct?

Reply:
If I don't add it I'm getting:
`Unsupported limittype 'None' in member 'schedulingControlsFlags' of structure 'VkPhysicalDeviceSchedulingControlsPropertiesARM'`

Reply:
Pulling...

Reply:
```
        if 'VkPhysicalDeviceSchedulingControlsPropertiesARM' in self.structs:
            self.structs['VkPhysicalDeviceSchedulingControlsPropertiesARM'].members['schedulingControlsFlags'].limittype = 'bitmask'
```
Fixed it for me :)

Reply:
Thanks!